From 7a9b7ec1ef194a01cf812119d9cb7dac9b32879b Mon Sep 17 00:00:00 2001
From: Mark Adler <madler@alumni.caltech.edu>
Date: Sun, 9 Feb 2020 22:07:34 -0800
Subject: [PATCH] Update beta level and date for zip bomb detection updates.

So that the fact that this is not an official Info-ZIP release can
be more easily detected.
---
 unzvers.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/unzvers.h b/unzvers.h
index 828f537..06a2514 100644
--- a/unzvers.h
+++ b/unzvers.h
@@ -21,8 +21,8 @@
 #  define UZ_BETALEVEL      "h03 BETA"
 #  define UZ_VERSION_DATE   "17 Apr 09"         /* internal beta version */
 #else
-#  define UZ_BETALEVEL      ""
-#  define UZ_VERSION_DATE   "20 April 2009"     /* official release version */
+#  define UZ_BETALEVEL      "b1"                /* bomb detection version 1 */
+#  define UZ_VERSION_DATE   "9 Feb 2020"        /* official release version */
 #  define RELEASE
 #endif
